utils/doclint/documentation.js

```diff
@@ -473,8 +473,10 @@ class Member {
       this.type.visit(visitor);
     for (const arg of this.argsArray)
       arg.visit(visitor);
+    for (const lang in this.langs.overrides || {})
```

Comment:
This is the actual change, which fixes the lang override comments in Python. The rest is drive-by ESLint auto fixes.